HTTP Error: 401 Authorization Required

Hello,

I've a strange issue, my token is well configured but deploy task fail with 401 Auth error.

Do you've any idea ? 

```
/action/node_modules/.bin/clever login --token *** --secret ***
/action/node_modules/.bin/clever link app_toto --alias app_toto
Your application has been successfully linked!
/action/node_modules/.bin/clever env set --alias app_toto version 1.0.0-alpha.26
Your environment variable has been successfully saved
/action/node_modules/.bin/clever deploy --alias app_toto
Error:  HTTP Error: 401 Authorization Required
Error: The process '/action/node_modules/.bin/clever' failed with exit code 1
```

Hello Guys,

Actually it appears to be a problem with git get remote info, when using the CLI with verbose there is what we get :

```
[STACKTRACE]
HttpError: HTTP Error: 401 Authorization Required
    at GitRemoteHTTP.discover (C:\Users\lil_r\AppData\Roaming\npm\node_modules\clever-tools\node_modules\isomorphic-git\index.cjs:6760:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.getRemoteInfo (C:\Users\lil_r\AppData\Roaming\npm\node_modules\clever-tools\node_modules\isomorphic-git\index.cjs:9557:20)
    at async Object.getRemoteCommit (C:\Users\lil_r\AppData\Roaming\npm\node_modules\clever-tools\src\models\git.js:62:23)
    at async Object.deploy (C:\Users\lil_r\AppData\Roaming\npm\node_modules\clever-tools\src\commands\deploy.js:23:30) {
  caller: 'git.getRemoteInfo',
  code: 'HttpError',
  data: {
    statusCode: 401,
    statusMessage: 'Authorization Required',
    response: 'Support for password authentication was removed on August 13, 2021.\n' +
      'Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes 
of authentication.\n'
  }
}
[/STACKTRACE]
[ERROR] HTTP Error: 401 Authorization Required
```

@Rhada did you managed to get through your issue ?

Hi @franky47, same issue here.

When trying locally, I also get the 401 error:

```
clever login --token [REDACTED] --secret [REDACTED]
➜  project git:(main) clever deploy --alias staging                                                                  
App is brand new, no commits on remote yet
New local commit to push is 2739273d6c6d5b96d11a54f6e395df6d6c78466f (from refs/heads/main)
Pushing source code to Clever Cloud…
[ERROR] HTTP Error: 401 Authorization Required
```

I tried to run `clever login` several times, with or without arguments (using the browser to login). I'm a bit stuck. Should I contact Clever Cloud support?

--- a/README.md
+++ b/README.md
@@ -9,6 +9,26 @@
 GitHub action to deploy your application to
 [Clever Cloud](https://clever-cloud.com).
 
+## Prerequisite
+
+⚠️ When creating an application on Clever Cloud, you have to choose
+between deploying "*from a local repository*" (using Clever CLI, Git
+or SFTP) or "*from a Github repository*" (using a webhook setup
+automatically by Clever Cloud). Only the first type of applications
+can be deployed using this Github action.
+
+In your project's `.clever.json`, if the `deploy_url` value starts
+with `https://github.com/`, your application is meant to be deployed
+"*from a Github repository*" only.
+If you try deploying it with this Github action, you will get the
+following message in your logs: `[ERROR] HTTP Error: 401 Authorization
+Required`.
+
+Currently (early 2023), the only workaround is to create a new
+application on Clever Cloud, that deploys "*from a local repository*",
+then remove the Clever Cloud webhook that has been created on your
+Github repository.
+
 ## Usage
 
 In your workflow file:

